### PR TITLE
Seperate Config File

### PR DESCRIPTION
**Pull request by TimothyWillard**
See issue #92.

### CODE REVIEW

File: lib/text.js [hunk 1]
```diff
@@ -83,7 +65,25 @@ function sendText(phone, message, carrier, region, cb) {
...
+/*  Overrides default config
+
+    Takes a new configuration object, which is
+    used to override the defaults
+
+    Params:
+      obj - object of config properties to be overriden
+*/
+function setConfig(obj) {
+  for (var prop in obj) {
```
Comment (typpo) [inline lib/text.js]: I think you could simplify this using `Object.assign` ([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)):

```
config = Object.assign(config, obj);
```

File: lib/text.js [hunk 2]
```diff
@@ -83,7 +65,25 @@ function sendText(phone, message, carrier, region, cb) {
...
+    if (obj.hasOwnProperty(prop)) {
+      config[prop] = obj[prop];
+    } 
+  }
+}
+
 module.exports = {
   send:       sendText,     // Send a text message
-  debug:      debug         // Enable or disable debug output
+  // debug:      debug,     No longer needed, see setConfig and config.js
```
Comment (typpo) [inline lib/text.js]: We can just remove this line :)

**Comment by TimothyWillard** [comment]
Made updates for those suggestions.

**Comment by farazfazli** [comment]
#106 is a README update for this change

**Pull request merged**